pkg/cloud/vsphere/assets/cloud-controller-manager-deployment.yaml

```
@@ -68,6 +68,8 @@ spec:
               value: {{ .globalCredsSecretName }}
             - name: ENABLE_ALPHA_DUAL_STACK
               value: "true"
+            - name: ADDITIONAL_NODE_LABELS
+              value: {{ .additionalLabels }}
```

Should we also have the if around this?


```suggestion
            {{- if ne .additionalLabels "" }}
            - name: ADDITIONAL_NODE_LABELS
              value: {{ .additionalLabels }}
            {{- end }}
```